.github/workflows/release.yml

```diff
@@ -24,6 +29,8 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+      with:
+        ref: ${{ inputs.ref }}
```

Comment:
I'm not sure if this will result in an error when the workflow is run directly or whether the default value will be used for this input, which is the behavior I want. Let's see...